global-news-app/src/assets/styles/global-styles.ts

```diff
@@ -1,5 +1,5 @@
-import { createGlobalStyle } from 'styled-components';
-import { theme } from './theme';
+import { createGlobalStyle } from "styled-components";
```

Comment:
make sure to have format on save enabled, and your code formatting is always consistent